2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,34 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+  while (list.firstChild) {
+    list.removeChild(list.firstChild);
+  }
+
+  todos.forEach((item) => {
+    let liEl = document.createElement("li");
+    liEl.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+    liEl.innerText = item.task;
+
+    let spanEl = document.createElement("span");
+    spanEl.className = "badge bg-primary rounded-pill";
+
+    let iEl1 = document.createElement("i");
+    iEl1.className = "fa fa-check";
+    iEl1["aria-hidden"] = "true";
+
+    let iEl2 = document.createElement("i");
```

Comment:
when you're creating variable names try and think what would make it easier for people who are unfamiliar with your code to understand- for example instead of `iEl1` and  `iEl2` names like `checkIcon` and `deleteIcon` would make the code more readable